glacier/src/exporters/state.ts

```diff
@@ -0,0 +1,24 @@
+import redux = require("redux");
+import JSZip = require("jszip");
+import {ModelState} from "../";
+import {Exporter, createSvgExporter} from "./";
+
+export function createStateExporter(store: redux.Store<ModelState>, uuid: number, libary: Buffer) {
```

Comment:
Also! This should probably be called the `zip` or `bundle` exporter, not the `state` exporter, since all exporters export the state in one way or another.